### PR TITLE
TCM-492 |  Add equipments section

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] Equipment Section added to workout screen

### Issue

[TCM-492](https://hyfnla.atlassian.net/browse/TCM-492)

### Video

https://user-images.githubusercontent.com/24737954/121410052-062d6080-c917-11eb-8d20-91b4afd1e295.mov

### CODE REVIEW

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt [hunk 1]
```diff
@@ -126,4 +132,11 @@ class WorkoutFragment : BaseFragment(R.layout.fragment_workout) {
     override fun onBackPressed() {
         viewModel.doTapBack()
     }
+
+    private fun listToString(list: List<String>): String {
```
Comment (devender197) [inline app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:136]: @onodude need suggestion on this function name and possibility as extension exit or not?

Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:136]: Couple things: 
- Needs an empty check or we'll crash
- Don't want to prepend "and" if it's just one item

Working well in other cases

Comment (VivekV95) [reply app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:136]: Also do we want an oxford comma or not? Currently we're not using one

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Just a couple things

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt (hunk 1 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:138]: [Done](https://github.com/hyfn/trx-consumer-android/pull/111/commits/458707ce45eb3979489509c914b58cc6b4f1abe6)

**Review by onodude — CHANGES_REQUESTED** [review]
@devender197 @VivekV95 would it be possible to mimic the changes on iOS with respect to the logic: https://github.com/hyfn/trx-consumer-ios/pull/194

@onodude Made similar to iOS @onodude [Done](https://github.com/hyfn/trx-consumer-android/pull/111/commits/2868fb736f3849604ec58b7d12a315815d16a122)

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt [hunk 2]
```diff
@@ -126,4 +134,13 @@ class WorkoutFragment : BaseFragment(R.layout.fragment_workout) {
...
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt:138]: [This](https://github.com/hyfn/trx-consumer-android/pull/111#pullrequestreview-679054052) should be possible here with same pattern, we have a `capitalized` extension we can use and I think we should be able to just call `joinToString` with a "," separator and inline everything like iOS is doing (if we don't care about adding an "and" at the end)

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Addressing @onodude 's request

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt (hunk 2 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt]: > @devender197 @VivekV95 would it be possible to mimic the changes on iOS with respect to the logic: [hyfn/trx-consumer-ios#194](https://github.com/hyfn/trx-consumer-ios/pull/194)

@onodude Made similar to iOS @onodude [Done](https://github.com/hyfn/trx-consumer-android/pull/111/commits/2868fb736f3849604ec58b7d12a315815d16a122)

**Comment by devender197** [comment]
> @devender197 @VivekV95 would it be possible to mimic the changes on iOS with respect to the logic: [hyfn/trx-consumer-ios#194](https://github.com/hyfn/trx-consumer-ios/pull/194)


@onodude Made similar to iOS @onodude [Done](https://github.com/hyfn/trx-consumer-android/pull/111/commits/2868fb736f3849604ec58b7d12a315815d16a122)

File: app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt (hunk 2 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/screens/workout/WorkoutFragment.kt]: addressed [here](https://github.com/hyfn/trx-consumer-android/pull/111#issuecomment-857895530)

**Comment by devender197** [comment]
@onodude only difference we have with iOS is that iOS don't have line below equipment section and android have it. do we need remove it on android also?

**Review by onodude — APPROVED** [review]
thanks!

**Pull request merged**